controllers/main.go

```diff
@@ -1822,7 +1828,7 @@ func (controller *MainController) Tickets(c web.C, r *http.Request) (string, int
 	}
 
 	c.Env["IsTickets"] = true
-	c.Env["Network"] = controller.getNetworkName()
+	c.Env["DCRDataURL"] = controller.DCRDataURL()
```

Comment:
Really good approach for consistency, but it seems like doing the `fmt.Sprintf` repeatedly is unnecessary.  Perhaps `DCRDataURL` could be a `string` field of `MainController` that gets set on construction of the controller.

Reply:
Nice idea. Implemented